.changeset/lazy-waves-build.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': minor
```

Comment:
Since this is a bug fix rather than a feature, should this be considered a `patch`?